packages/boxel/addon/components/boxel/tab-bar/index.css

```diff
@@ -19,13 +52,12 @@
   https://css-tricks.com/bold-on-hover-without-the-layout-shift/
 */
 
-.boxel-tab-bar__item > a {
-  display: inline-flex;
+.boxel-tab-bar__item-text {
+  display: flex;
   flex-direction: column;
-  padding: var(--boxel-sp-xs) var(--boxel-sp-lg);
 }
 
-.boxel-tab-bar__item > a::after {
+.boxel-tab-bar__item-text::after {
   content: attr(data-text);
   content: attr(data-text) / "";
```

Comment:
Do we need both these content properties?

Reply:
hmm well Firefox [doesn’t support](https://developer.mozilla.org/en-US/docs/Web/CSS/content#browser_compatibility) the `/` for fallback but I tried removing `data-text` and it was fine (with the jumping of course) so I guess not…??? Thanks